src/types.rs

```diff
@@ -277,6 +277,7 @@ pub struct ObjectInfo {
 
     /// Identifier of the code file.
     #[serde(skip_serializing_if = "is_default")]
+    #[serde(default)]
```

Comment:
_nit_: For `Option<_>` this is not required, but it's good to have this here explicitly 👍